Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Bundle app source
+COPY feup-exchange/ .
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
```

Comment:
I'm not very familiar with Docker, but shouldn't we have something like a `Dockerfile.prod` instead of having this line commented here?

Reply:
Can confirm this is usually the case. I think it's even more common to have a `Dockerfile.dev` for development, followed by a `Dockerfile` or `Dockerfile.prod` for production environment :smile:

Reply:
Ok cool, I'm gonna change this.

Reply:
I left the Dockerfile as the development one and only created the .prod. 
This way developing (which will be most of the docker activities) will be simpler, and whenever we want to deploy we'll just change the file names and do so.

Reply:
@pedromsfernandes @andrefmrocha

Reply:
Just a question, is this PR meant to have hot reloading or not?